base/common/src/main/java/com/netscape/certsrv/profile/ProfileResource.java

```diff
@@ -20,7 +20,9 @@ public interface ProfileResource {
     @ACLMapping("profiles.list")
     public Response listProfiles(
             @QueryParam("start") Integer start,
-            @QueryParam("size") Integer size);
+            @QueryParam("size") Integer size,
+            @QueryParam("visible") Boolean visible,
+            @QueryParam("enable") Boolean enable);
```

Comment:
It would be nice if we can add options into `ProfileFindCLI` so we can test these params from CLI, but it's not required.

Reply:
Added. I have added 3 option but they work slightly differently.
The new options for the command ca-profile-find are:

- `--visible` shows only visible profiles (this is the default behaviour for non admin users);
- `--enable` shows only the enabled profiles;
- `--enableBy <username>` shows the profiles enabled by the provided
      user.